src/main/java/com/openlattice/edm/collection/EntityTypeCollection.java

```diff
@@ -0,0 +1,103 @@
+package com.openlattice.edm.collection;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.authorization.securable.AbstractSchemaAssociatedSecurableType;
+import com.openlattice.authorization.securable.SecurableObjectType;
+import com.openlattice.client.serialization.SerializationConstants;
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+
+import java.util.*;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public class EntityTypeCollection extends AbstractSchemaAssociatedSecurableType {
+
+    private LinkedHashSet<CollectionTemplateType> template;
```

Comment:
Maybe I'm missing something, but why don't we use only entity type ids here?

Reply:
There's more metadata that needs to be tracked for each of these (they basically replace app types) -- also there will likely be multiple template types of the same entity type (i.e. a collection that involves multiple people entity sets, like arrestees and officers)

Reply:
👍